chain/indexer/Cargo.toml

```
@@ -18,6 +18,7 @@ serde = { version = "1", features = [ "derive" ] }
 serde_json = "1.0.55"
 tokio = { version = "1.1", features = ["time", "sync"] }
 
+indexer-primitives = { path = "../indexer-primitives" }
```

```suggestion
near-indexer-primitives = { path = "../indexer-primitives" }
```

and move it down the list maintaining the alphabetical order if rustfmt does not do it automatically.